store/migration/dev/20221027000000##approval_instance.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE approval_instance ( 
+    id SERIAL PRIMARY KEY,
+    row_status row_status NOT NULL DEFAULT 'NORMAL',
+    created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    issue_id INTEGER NOT NULL REFERENCES issue (id),
+    requester_id INTEGER NOT NULL REFERENCES principal (id),
+    approver_id INTEGER NOT NULL REFERENCES principal (id),
+    type TEXT NOT NULL CHECK (type LIKE 'bb.application.%'),
```

Comment:
why bb.application instead of reusing bb.plugin?

What's the specific type value we would store?

"bb.application.xxx" doesn't seem to relate with the table name approval_instance.

e.g. Policy table has a type starting with "bb.policy."